lib/cfe_internal_hub.cf

```diff
@@ -202,8 +202,21 @@ bundle agent cfe_internal_database_cleanup_consumer_status (row_count)
 {
 
   vars:
+
+    any::
+
+      "status_table_name" -> { "ENT-4331" }
+        string => "__status";
+
+    cfengine_3_7|cfengine_3_10|cfengine_3_12_0|cfengine_3_12_1::
+
+      "status_table_name" -> { "ENT-4331" }
+        string => "status";
```

Comment:
@nickanderson So you are relying on the ordering of vars promises here to overwrite. Is this something we do in other places? Should we document that it's okay, here: https://docs.cfengine.com/docs/3.12/reference-promise-types-vars.html ?

Reply:
I thought I responded to this before. Yes, I am relying on the order of vars and over-writing. It's documented on the page you reference under the policy attribute. Happy to improve the docs if you have a suggestion as to specifically where should include something else.

Reply:
The behaviour is not really different from other promise types.  Normal Ordering https://docs.cfengine.com/docs/3.12/reference-language-concepts-normal-ordering.html#agent-evaluation-step notes:

The order of promises within one of the above types follows their top-down ordering within the bundle itself. The order may be overridden by making a promise depend on a class that is set by another promise, or by using the depends_on attribute in the promise.

So you may want to sprinkle your doc pr in a couple places

https://docs.cfengine.com/docs/3.12/reference-language-concepts-variables.html
https://docs.cfengine.com/docs/3.12/reference-language-concepts-normal-ordering.html
https://docs.cfengine.com/docs/3.12/reference-promise-types-vars.html

Reply:
https://github.com/cfengine/documentation/pull/2064